packages/fxa-payments-server/src/lib/amplitude.test.ts

```diff
@@ -1,39 +1,16 @@
-import { Middleware, MiddlewareAPI, Dispatch, AnyAction } from 'redux';
```

Comment:
The big thing here is that I converted the amplitude-middleware switch statement into a set of plain functions. I tried to adapt the existing tests with as few changes as possible to try to prove that the new functions are basically equivalent